cmake/checks/check_02_compiler_features.cmake

```diff
@@ -126,8 +111,7 @@ CHECK_CXX_COMPILER_BUG(
 #
 # - Matthias Maier, rewritten 2012
 #
-IF(NOT(CMAKE_CXX_COMPILER_ID MATCHES "Intel" AND
-       DEAL_II_HAVE_CXX14_CONSTEXPR))
```

Comment:
Don't you still need this (with `DEAL_II_CXX14_CONSTEXPR_BUG_OK`)?

Reply:
Or you just wanted to simplify by disabling for all `Intel` compilers?

Reply:
Correct. Fixed.